extensions/adobe/experience/decisioning/activity-detail.schema.json

```diff
@@ -60,6 +60,14 @@
           "description": "The reference to a fallback option that is used when decisioning in the context of this activity does not qualify any of the regular options (this typically happens when hard constraints are applied). The value is the URI (@id) of the fallback option that is referenced.",
           "meta:titleId": "activity-detail##xdm:fallback##title##82451",
           "meta:descriptionId": "activity-detail##xdm:fallback##description##3941"
+        },
+        "xdm:matchedSurfaces": {
+          "type": "array",
+          "items": {
+            "type": "string"
```

Comment:
@krishnanbalasubramanyam - your example shows URI format. so do you want to add `"format": "uri-reference"` here, or just open `string` type without any format enforcement will work?

Reply:
@anandphatak At this point, it is ok to leave it as an open string type.

Reply:
Ok. At a later point you may not be able to add it because if any data is ingested against it may not comply to the uri format. If you are ok with it then we can approve it